discordSuperUtils/imaging.py

```diff
@@ -360,6 +362,123 @@ async def create_leveling_profile(
         result_bytes.seek(0)
         return discord.File(result_bytes, filename="rankcard.png")
 
+    async def create_spotify_card(
+        self,
+        spotify_result: discord.Spotify,
+        font_path: str = None
+    ) -> discord.File:
+        """
+        |coro|
+
+        Creates a welcome image for the member and returns it as a discord.File.
+
+        :param discord.Spotify spotify_result: The Spotify activity.
+        :param str font_path: The font path, uses the default font if not passed.
+        :return: The discord file.
+        :rtype: discord.File
+        """
+
+        result_bytes = BytesIO()
+
+        track_background_image = Image.open(
+            ImageManager.load_asset(
+                "spotify_template.png"
+            )
+        )
+        
+        album_image = Image.open(
+                requests.get(
+                    spotify_result.album_cover_url, 
+                    stream=True
+                ).raw
+            ).convert('RGBA')
+        
+        font_path = font_path if font_path else self.load_asset("font.ttf")
+        
+        # Fonts
+        title_font = ImageFont.truetype(font_path, 16)
+        artist_font = ImageFont.truetype(font_path, 14)
+        album_font = ImageFont.truetype(font_path, 14)
+        start_duration_font = ImageFont.truetype(font_path, 12)
+        end_duration_font = ImageFont.truetype(font_path, 12)
+
+        # Positions
+        title_text_position = 150, 30
+        artist_text_position = 150, 60
+        album_text_position = 150, 80
+        start_duration_text_position = 150, 122
+        end_duration_text_position = 515, 122
+
+        duration = time.strftime(
+            "%H:%M:%S", 
+            time.gmtime(
+                spotify_result.duration.total_seconds()
+                )
+            )
+        
+        # Draws
+        draw_on_image = ImageDraw.Draw(
+            track_background_image
+        )
+
+        draw_on_image.text(
+            title_text_position, 
+            spotify_result.title, 
+            'white', 
+            font=title_font
+        )
+
+        draw_on_image.text(
+            artist_text_position, 
+            f'by {spotify_result.artist}', 
+            'white', 
+            font=artist_font
+        )
+
+        draw_on_image.text(
+            album_text_position, 
+            spotify_result.album, 
+            'white', 
+            font=album_font
+        )
+
+        draw_on_image.text(
+            start_duration_text_position, 
+            '0:00', 
```

Comment:
perhaps instead of 0:00, it could show the current second of the song?
for example, this:
```py
current_second = spotify_result.duration - (spotify_result.end - spotify_result.start)
```
keep in mind that is a timedelta, so you might have to format it. (You can use the default format, of course.)

Reply:
Good alternative!